sdk/core/perf-tests/core-rest-pipeline/test/index.spec.ts

```diff
@@ -1,8 +1,23 @@
 import { PerfProgram, selectPerfTest } from "@azure/test-utils-perf";
 import { BearerTokenAuthenticationPolicyChallengeTest } from "./bearerTokenChallengeAuthenticationPolicy/wwwChallenge.spec";
+import { CoreHTTPTest } from "./core-http.spec";
+import { CoreRestPipelineTest } from "./core-rest-pipeline.spec";
+import { FetchTest } from "./fetch.spec";
+import { HttpRequestTest} from "./http-request.spec";
+import { UndiciRequestTest} from "./undici-request.spec";
+// import { NodeFetchTest } from "./node-fetch.spec";
```

Comment:
Can we figure out what is wrong with `node-fetch` before merging this?  I'd say either fix the test or remove it from the PR completely.

Reply:
Deleted the test for now. Logging an issue 

> node-fetch is an ESM-only module - you are not able to import it with require. We recommend you stay on v2 which is built with CommonJS unless you use ESM yourself. We will continue to publish critical bug fixes for it.

And v2 needs support for https server than just http that this PR has, which would need self-signed certificates, will take that as a separate PR in order not to block the already working tests.